apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/SpringConfiguration.java

```diff
@@ -59,12 +59,14 @@
 import org.springframework.orm.jpa.JpaTransactionManager;
 import org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean;
 import org.springframework.orm.jpa.support.PersistenceAnnotationBeanPostProcessor;
+import org.springframework.retry.annotation.EnableRetry;
 import org.springframework.scheduling.annotation.EnableScheduling;
 
 /** The main Spring {@link Configuration} for the Blue Button API Backend application. */
 @Configuration
 @ComponentScan(basePackageClasses = {ServerInitializer.class})
 @EnableScheduling
+@EnableRetry(proxyTargetClass = true)
```

Comment:
As far as I understand it, if we were to use interfaces for each provider class rather than the concrete implementation we wouldn't need to specify `proxyTargetClass` here (the `Proxy` will explicitly subclass the class with any `@Retryable`-decorated methods).

The side effect of doing it this way is that all provider classes need to be non-`final` so that the generated `Proxy`s _can_ subclass them properly.